dotnet/src/Connectors/Connectors.AI.HuggingFace/TextGeneration/HuggingFaceTextGenerationService.cs

```diff
@@ -29,6 +29,21 @@ public sealed class HuggingFaceTextGenerationService : ITextGenerationService
     private readonly string? _apiKey;
     private readonly Dictionary<string, object?> _attributes = new();
 
+    #region BreakingGlass Properties
+    /// <summary>
+    /// Model Id.
+    /// </summary>
+    public string? ModelId => this.GetModelId();
```

Comment:
This isn't breaking glass, this get's set by the client.

The value are available via `IAIService.Attributes` already so no need to add these here.

Reply:
Removed.